contracts/strategies/allocation/hackathon-qv/README.md

```diff
@@ -0,0 +1,37 @@
+Spec: Hackathon Quadratic Voting (QV)
+---------------------------------
+
+## Overview 
+Quadratic Voting is a popular method for democratic decision-making. This strategy is designed to facilitate voting for the winners of a hackathon. An organizer is able to programmatically say who is an eligible recipient and decide to gate voting using Passport. The pool funds are distributed in tiers based on each recipients place in the quadratic vote. 
+
+## Spec
+### Recipient logic
+In this strategy, the pool admin uses an Allo identity and an EAS attestation to indicate eligible recipients. Any Allo identities that have the admin-designated attestation are eligible recipients. 
+- `registerRecipient` - function for pool admin to indicate the attestation that a recipient must have to be eligible. 
+- `updateRecipientList` — function that anyone can call to pull the latest list of eligible recipients
+- `updateRecipient` — function that the identity admin can use to add metadata about their project
+
+### Voter eligibility logic
+Pool admins are able configure Passport to gate voting. A voter must meet the community's Passport requirements in order to be marked as eligible. 
+
+### Voting logic
+In this strategy, only the eligible wallets are able to vote. Votes can only be cast during a voting window that is set by the pool admin. Each voter gets a budget of "voice credits", which they can use to "purchase" votes on as many recipients as they would like. The number of votes each voter can give to a recipient is equal to the square root of the number of voice credits they spend on that specific recipient. Votes can be fractional. 
```

Comment:
Do these "voice credits" just live in the contract and only a representation of their voting power or ERC20 type eligibility?

Side thought: VOICE token on PGN